README.md

```diff
@@ -1 +1,24 @@
-# hello-react-rails-app
\ No newline at end of file
+# README
```

Comment:
- [ ] On your readme, please, implement the [Microverse readme template](https://github.com/microverseinc/readme-template) so your project is well documented. Please, ensure that you include explicit steps on how to run your project.  

All the sections of the template are required except for the "tests" and "live demo" if not required by the project.